project/MiMaFilters.scala

```diff
@@ -7,4 +7,8 @@ object MiMaFilters {
     ProblemFilters.exclude[DirectMissingMethodProblem]("scala.caps.unsafeUnbox"),
   )
   val TastyCore: Seq[ProblemFilter] = Seq()
+  val Interfaces: Seq[ProblemFilter] = Seq(
+    ProblemFilters.exclude[MissingClassProblem]("dotty.tools.dotc.interfaces.DiagnosticRelatedInformation"),
+    ProblemFilters.exclude[ReversedMissingMethodProblem]("dotty.tools.dotc.interfaces.Diagnostic.diagnosticRelatedInformation")
+  )
```

Comment:
I don't think this is necessary, since the baseVersion is 3.3.0-RC1, mima is set to run in backwards mode which lets us add new APIs without having to filter them, I checked that `scala3-interfaces/mimaReportBinaryIssues` did not fail if I removed this.

Reply:
For me this fails with
```scala
[error] scala3-interfaces: Failed binary compatibility check against org.scala-lang:scala3-interfaces:3.2.2! Found 1 potential problems
[error]  * abstract method diagnosticRelatedInformation()java.util.List in interface dotty.tools.dotc.interfaces.Diagnostic is present only in current version
[error]    filter with: ProblemFilters.exclude[ReversedMissingMethodProblem]("dotty.tools.dotc.interfaces.Diagnostic.diagnosticRelatedInformation")
[error] java.lang.RuntimeException: Failed binary compatibility check against org.scala-lang:scala3-interfaces:3.2.2! Found 1 potential problems
[error]         at scala.sys.package$.error(package.scala:30)
[error]         at com.typesafe.tools.mima.plugin.SbtMima$.reportModuleErrors(SbtMima.scala:89)
[error]         at com.typesafe.tools.mima.plugin.MimaPlugin$.$anonfun$projectSettings$2(MimaPlugin.scala:36)
[error]         at com.typesafe.tools.mima.plugin.MimaPlugin$.$anonfun$projectSettings$2$adapted(MimaPlugin.scala:26)
[error]         at scala.collection.Iterator.foreach(Iterator.scala:943)
[error]         at scala.collection.Iterator.foreach$(Iterator.scala:943)
[error]         at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
[error]         at com.typesafe.tools.mima.plugin.MimaPlugin$.$anonfun$projectSettings$1(MimaPlugin.scala:26)
[error]         at com.typesafe.tools.mima.plugin.MimaPlugin$.$anonfun$projectSettings$1$adapted(MimaPlugin.scala:25)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error]         at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error]         at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error]         at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error]         at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error]         at sbt.Execute.work(Execute.scala:291)
[error]         at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error]         at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error]         at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error]         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error]         at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error]         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error]         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error]         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error]         at java.lang.Thread.run(Thread.java:748)
```
And this also failed in the CI. That's why I added these exceptions

Reply:
Does `show scala3-interfaces/mimaCheckDirection` outputs `backward`? If not, you may need to reload the sbt build / rebase